db/seeds.rb

```diff
@@ -6,16 +6,20 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+User.create(email: "email@example.com", password: "password1234")
```

Comment:
consider using a `create!` here to ensure the seed is noisy if validation fails.